libs/blog/ad-banner/ui/src/lib/infinite-slider-directive/al-infinite-slider.directive.ts

```diff
@@ -33,9 +33,15 @@ import { debounceTime, interval, tap } from 'rxjs';
   standalone: true,
 })
 export class AlInfiniteSliderDirective {
-  readonly alInfiniteSliderOf = input.required<unknown[]>();
-  readonly msPerSlide = input<number>(7000);
-  readonly msPerAnimation = input<number>(1000);
+  readonly slidesElements = input.required<unknown[]>({
+    alias: 'alInfiniteSliderOf',
+  });
+  readonly msPerSlide = input<number>(7000, {
+    alias: 'alInfiniteSliderMsPerSlide',
+  });
+  readonly msPerAnimation = input<number>(1000, {
+    alias: 'alInfiniteSliderMsPerAnimation',
+  });
```

Comment:
Same story. Proctected

Reply:
that's an input so we need to keep it public